Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -920,27 +925,30 @@ def enforcement_boundary_create_command(
         providers=providers,  # type: ignore
         ingress_services=[{"port": port, "proto": proto}],
     )
-    enforcement_boundary_json = ""
+    enforcement_boundary_json = {}
     try:
         enforcement_boundary = client.enforcement_boundaries.create(enforcement_boundary_rule)  # type: ignore
-        enforcement_boundary_href = enforcement_boundary.href
         enforcement_boundary_json = enforcement_boundary.to_json()
-        enforcement_boundary_json["href"] = enforcement_boundary_href
-        enforcement_boundary_json["already_exists"] = False
+        enforcement_boundary_json["already_exists"] = False  # type: ignore
```

Comment:
Remove the `already_exists` parameters. 

There's also no need to duplicate behaviour between the two success cases here - we can call the `prepare_output` function and `to_json()` below the try block.

The best approach would probably be to genericize this pattern of POST/GET by ref on fail and bubble any exceptions.